libglog.pc.in

```diff
@@ -1,10 +1,10 @@
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
+prefix=@CMAKE_INSTALL_PREFIX@
```

Comment:
Doesn't this break autoconf support?

Reply:
Maybe it's OK now.

Reply:
CI failed because of the lack of <sys/wait.h> in MinGW toolchain.